test/integration_test_helper.rb

```diff
@@ -28,8 +29,19 @@
   Capybara::Poltergeist::Driver.new(app, opts)
 end
 
-Capybara.default_max_wait_time = 30
-Capybara.javascript_driver = :poltergeist
+Selenium::WebDriver::Chrome.driver_path = File.join(Rails.root, 'node_modules', '.bin', 'chromedriver')
```

Comment:
worth noting that in the past, I had an issue where npm chromedriver was newer then the chrome (or chromium) that was installed on slaves, but was too old for the versions you found in fedora today.

Reply:
There is now `TESTDRIVER_PATH` env variable to allow setting this per environment if necessary.